src/darwin.rs

```diff
@@ -7,6 +7,7 @@ use libc::{
 
 use crate::MemoryStats;
 
+/// Returns the data as bytes
```

Comment:
Since the `src/{darwin, linux, windows}.rs` files are technically internal implementation files that should all do the same thing anyways, I don't think these really need to be individually documented